plugin/src/main/java/io/jenkins/plugins/casc/BaseConfigurator.java

```diff
@@ -298,7 +301,8 @@ public T check(CNode c, ConfigurationContext context) throws ConfiguratorExcepti
      */
     protected void configure(Mapping config, T instance, boolean dryrun, ConfigurationContext context) throws ConfiguratorException {
         final Set<Attribute<T,?>> attributes = describe();
-        for (Attribute<T,?> attribute : attributes) {
+        List<Attribute<T, ?>> sortedAttributes = sortAttributes(attributes);
```

Comment:
As this is the protected `configure` block I feel safe to say that other plugins providing configurators will also be sorted according to the ordinal.